packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -131,8 +263,8 @@ import 'inherited_notifier.dart';
 ///    distributes key events to focused nodes.
 ///  * [FocusTraversalPolicy], an object used to determine how to move the focus
 ///    to other nodes.
-///  * [DefaultFocusTraversal], a widget used to configure the default focus
-///    traversal policy for a widget subtree.
+///  * [FocusTraversalGroup], a widget that groups together and imposes a
```

Comment:
Same comment here about the See Alsos

Reply:
OK, I removed some unnecessary (or less important) references, and reorganized it a bit.